package.json

```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "bluebird": "2.2.2",
     "bunyan": "1.0.0",
-    "fxa-content-server-l10n": "git://github.com/mozilla/fxa-content-server-l10n.git#4168ef3cb7",
+    "fxa-content-server-l10n": "git://github.com/mozilla/fxa-content-server-l10n.git",
```

Comment:
I think we might be able to set this free. It was locked down before because this repo didn't "control" its own strings so we locked it to a known good commit. Now, as long as we follow the workflow in the readme we should be ok. *As long as we don't run npm install on and outdated version of this repo* (but that's not how we roll... literally)